source/MaterialXRender/Util.cpp

```diff
@@ -239,7 +239,7 @@ unsigned int getUIProperties(InputPtr input, const string& target, UIProperties&
 }
 
 void createUIPropertyGroups(DocumentPtr doc, const VariableBlock& block, UIPropertyGroup& groups,
-                            UIPropertyGroup& unnamedGroups, const string& pathSeparator)
+                            UIPropertyGroup& unnamedGroups, const string& pathSeparator, bool showAllInputs)
```

Comment:
This should not be added in as I removed this when fixing up this code :).

Reply:
Will revert.